### PR TITLE
slack

### PR DESCRIPTION
**Pull request by valeIT**

### CODE REVIEW

File: _posts/2020-01-10-communication-on-slack.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+type: post
+published: true
+status: publish
+categories: [General]
+image:
+image2:
+author: Valentino Urbano
+---
+
+At work whenever we don't need emails we use Slack. You can say we use it for everything. I guess most companies do the same nowadays. Some times though it is used too much and for the wrong use cases. The secret in effectively using Slack is knowing what it is for and what it is not.
```
Comment (MsSlacker) [inline _posts/2020-01-10-communication-on-slack.md]: Sometimes should be together, right?

File: _posts/2020-01-10-communication-on-slack.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+- Chat
+
+Slack is a chat app and it is definitely not where to keep your documentation or where to document your processes. If you know that you will need something from Slack at a later date extract it in a Google Docs and put the link in slack instead. It is fine if the conversation happens in Slack, but once is finalized the result should be recorded somewhere else where it can be accessed (Google Docs, Jira,..)
+
+- Groups
+
+Groups are great. When you have a discussion about something that relates to the whole project or part of it or in any other case that interests more than 1 person create a group for it. Try to avoid using direct messages if it's not strickly necessary so other people can take advantage of what was discussed and can keep up to date without having to always ask the same questions. As before make sure to finally extract the resulting documentation out of Slack for later use.
```
Comment (MsSlacker) [inline _posts/2020-01-10-communication-on-slack.md]: strickly - > strictly?

File: _posts/2020-01-10-communication-on-slack.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+3. Ping when a new Pull Request is open
+4. Ping when a new Pull Request is merged
+...
+
+- Calendar
+
+It integrates well with Google calendar to let other people know if you're in a meeting or not.
+
+- Status
+
```
Comment (MsSlacker) [inline _posts/2020-01-10-communication-on-slack.md:44]: tent - > tend

**Review by MsSlacker — CHANGES_REQUESTED** [review]

File: _posts/2020-01-10-communication-on-slack.md (hunk 1 above)
Comment (valeIT) [reply _posts/2020-01-10-communication-on-slack.md]: sure

**Review by MsSlacker — APPROVED** [review]

**Pull request merged**